test/inspect-test.js

```diff
@@ -72,6 +74,16 @@ tape('v8 inspect', (t) => {
     t.ok(/.other-key=[^\n]*<String: "ohai">/.test(lines),
          '.other-key property');
 
+    const arrayMatch = lines.match(
+    /.array=(0x[0-9a-f]+):<Array: length=6>/);
```

Comment:
Can you indent by 4 spaces here and elsewhere?  (Better, indent but break after the `=`.)